src/scss/7-components/_main-header.scss

```diff
@@ -7,23 +7,22 @@
   --p-main-header-border-color: rgb(0,0,0, 0.1);
 
   position:sticky; z-index:999; inset-block-start:0; background-color:hsl(var(--p-body-bg-color));
-  border-block-end: solid 1px var(--p-main-header-border-color);
-  padding-inline: clamp(1.25rem, 4vw, 120rem); 
-  
-  &.is-docs {
-    padding-inline: pxToRem(32);
+  padding-inline:pxToRem(32); border-block-end: solid pxToRem(1) var(--p-main-header-border-color);
+  min-block-size:pxToRem(73);
 
+  &.is-special-padding {
```

Comment:
We only use the `main-header` in two places. One is `Main.svelte`, and the other is `Docs.svelte`. 

Instead of having one extra class per each, wouldn't it make more sense for one to use the default padding, and the other overrides it?

Reply:
Similar to what it was before actually

Reply:
@TGlide The particular padding for the homepages, the calculated dynamic, isn't so healthy for maintaining.

this is the reason why I separated it only while using special class.

Reply:
> isn't so healthy for maintaining
Why?